pkg/proxy/options_test.go

```diff
@@ -133,7 +133,6 @@ func TestRuleConfig(t *testing.T) {
 	require.Len(t, rules, 1)
 	require.Len(t, rules[0].PreFilter, 1)
 	require.Len(t, rules[0].Checks, 0)
-	require.Len(t, rules[0].Updates, 0)
```

Comment:
Why this change?

Reply:
Probably removed it while working on changing the name and forgot to go back and fix it

Reply:
Fixed